_layouts/home.html

```diff
@@ -220,6 +221,50 @@ <h2>Events</h2>
     </div>
 
     <script src="{{site.baseurl}}/assets/js/main.js"></script>
+    <script>
+        $.ajax({
+        method: "get",
+        url: "https://ieee.nitk.ac.in/blog/search.json",
+        success: function (result) {
+        add_blogs(result)
+        }
+        });
+
+        function add_blogs(result) {
+            blog = result;
+            var i = 1;
+            var html_str = '';
+            if (i<=3){
+            for (items in blog ){
+                if (i > 3) { break; }
+
+                str = blog[items].author;
+                str = str.split(" ").join("");
+                author_img_url = `https://ieee.nitk.ac.in/blog/assets/img/authors/${str}.jpeg`;
+
+                html_str += `<div class="card-deck"><div class="animate-box"><article role="article" itemscope="itemscope" itemtype="http://schema.org/BlogPosting" itemprop="blogPost">`;
```

Comment:
It looks like the `animate-box` class causes the blogs to render in with opacity 0. Try removing the class and see if the blogs are loading in properly.